i18n.js

```diff
@@ -32,6 +32,9 @@ function setPageLanguage(lang) {
   else if (lang === "fr-FR")
     langObj = frFR;
 
+  else if (lang === "du-NL"
```

Comment:
Dank je wel, maar da is een fout in jij code!

You forgot to add the closing bracket :)